docs/content/docs/dev/testing.md

```diff
@@ -41,26 +41,33 @@ flutter test
 
 ### Infrastructure
 
-You need a fresh [`synapse` matrix backend](https://matrix-org.github.io/synapse/latest/) with the following settings included (in the `homeserver.yaml`):
+You need a fresh [`synapse` matrix backend](https://matrix-org.github.io/synapse/latest/).
 
-```yaml
+Please change `bind_addresses` of `listeners` from `['::1', '127.0.0.1']` to `['0.0.0.0']`, that means any address and allows remote connection (non-localhost).
+
+Please turn off the public profile of server firewall, that allows synapse to accept the request from external. If you are running `Ubuntu` linux, you can use `gufw` app to do it.
```

Comment:
Yes, correct

Reply:
Please review this comment too.
I think it is needed to insert that comment.
`To avoid the change of server URL under VMWare, you can use NAT mode not Bridged mode as network.`